### PR TITLE
NewsML-G2 v2.34

### PR DESCRIPTION
**Pull request by bquinn**
Changes in version 2.34:

* http://dev.iptc.org/G2-CR00218-increase-nesting-of-related (fixes iptc/news-architecture-dev#144)
* http://dev.iptc.org/G2-CR00219-add-datamining-element-to-rightsinfo (fixes iptc/news-architecture-dev#136)
* Updating dev schema to be based on version 2.34. Fixes iptc/news-architecture-dev#146

### CODE REVIEW

**Comment by davecompton** [comment]
Hi @bquinn - just checking: So have all the CR00218 & CR00219 changes been applied for this pull - looks like so, from the above. Thanks.

**Comment by bquinn** [comment]
> Hi @bquinn - just checking: So have all the CR00218 & CR00219 changes been applied for this pull - looks like so, from the above. Thanks.

Actually no, I still need to commit the dataMining changes. I'm trying to get the RightsML example to validate properly.

**Comment by bquinn** [comment]
> > Hi @bquinn - just checking: So have all the CR00218 & CR00219 changes been applied for this pull - looks like so, from the above. Thanks.
> 
> Actually no, I still need to commit the dataMining changes. I'm trying to get the RightsML example to validate properly.

It's working and committed now! Please take a look.

File: specification/NewsML-G2_2.34-spec-All-Power.xsd
Comment (davecompton) [inline specification/NewsML-G2_2.34-spec-All-Power.xsd:1]: Still shows v2.33 in comments?

File: tests/schema_versions/NewsML-G2_2.34-spec-All-Power.xsd
Comment (davecompton) [inline tests/schema_versions/NewsML-G2_2.34-spec-All-Power.xsd:1]: Still shows v2.33 in comments?

**Comment by bquinn** [comment]
good point! I just fixed it.

File: specification/NewsML-G2_2.34-spec-All-Power.xsd
Comment (davecompton) [inline specification/NewsML-G2_2.34-spec-All-Power.xsd:1]: Still shows: Date of creation of this XML Schema document revision: 2023-08-25

File: tests/schema_versions/NewsML-G2_2.34-spec-All-Power.xsd
Comment (davecompton) [inline tests/schema_versions/NewsML-G2_2.34-spec-All-Power.xsd:1]: Still shows: Date of creation of this XML Schema document revision: 2023-08-25

File: releases/2.34/specification/XML-Schema-Doc-Power/NewsML-G2_2.33-spec-All-Power.html
Comment (davecompton) [inline releases/2.34/specification/XML-Schema-Doc-Power/NewsML-G2_2.33-spec-All-Power.html:1]: Filenames show 2.33.

File: examples/LISTING_4_Multiple_Renditions_of_a_Video_in_NewsML-G2.xml
Comment (davecompton) [inline examples/LISTING_4_Multiple_Renditions_of_a_Video_in_NewsML-G2.xml:1]: Odd ^M in this file?

File: releases/2.34/specification/XML-Schema-Doc-Power/NewsML-G2_2.33-spec-All-Power.html
Comment (bquinn) [reply releases/2.34/specification/XML-Schema-Doc-Power/NewsML-G2_2.33-spec-All-Power.html:1]: Yep I haven't updated the XML Spy schema docs yet. I mentioned that in a commit message but github might have buried it somewhere.

File: examples/LISTING_4_Multiple_Renditions_of_a_Video_in_NewsML-G2.xml
Comment (bquinn) [reply examples/LISTING_4_Multiple_Renditions_of_a_Video_in_NewsML-G2.xml:1]: I think that has been around for a long time. Fixed now.

**Pull request merged**